x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -23,6 +23,10 @@ import {
 } from './bottom_bar';
 import { useShowTimeline } from '../../../common/utils/timeline/use_show_timeline';
 import { gutterTimeline } from '../../../common/lib/helpers';
+import { useSourcererScope } from '../../../common/containers/sourcerer';
+import { OverviewEmpty } from '../../../overview/components/overview_empty';
+import { ENDPOINT_METADATA_INDEX } from '../../../../common/constants';
+import { useFetchIndex } from '../../../common/containers/source';
 
 /* eslint-disable react/display-name */
```

Comment:
fwiw, this was originally added because we weren't setting the react display name and we added a linter rule for it and disabled it. However, this linter rule where possible should be removed. You don't have to do it here, but just mentioning it. Information is here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md

Basically it helps us debug React applications. (Again background info, you don't have to do anything in this PR)